web_external/js/views/body/Panel.js

```diff
@@ -0,0 +1,55 @@
+minerva.views.Panel = minerva.View.extend({
+    /**
+     * The panel view isn't meant to be instantiated on it's own.
+     **/
+    events: {
+        'click .icon-cancel': 'removePanel',
+        'show.bs.collapse': 'expandPanel',
+        'hide.bs.collapse': 'collapsePanel'
+    },
+
+    getSessionView: function () {
+        return this.parentView.parentView;
+    },
+
+    getSessionModel: function () {
+        return this.getSessionView().model;
+    },
+
+    /**
+     * Upon confirmation, physically removes the DOM element while
+     * storing the change in the session attributes. This also
+     * enables the save button.
+     **/
```

Comment:
Nope, unless you were to manually edit the session.json. It seems like having a dropdown with available panels is something @aashish24 wants at some point - so that could add that functionality.